scripts/regenerate.sh

```diff
@@ -15,14 +16,10 @@
 
 set -eu -o pipefail
 
-WORKDIR=$(mktemp -d)
-
-function finish {
-  rm -rf "$WORKDIR"
-}
-trap finish EXIT
+WORKDIR="/tmp/grpc-go-tools"
+mkdir -p "${WORKDIR}"
 
-./scripts/install-protoc.sh "${WORKDIR}"
+"./$(dirname "${0}")/install-protoc.sh" ${WORKDIR}
```

Comment:
I think you want to remove `./` at the start?  Try executing as `/abs/path/to/scripts/regenerate.sh` and I think it will fail?

Reply:
Yea, for whatever reason i've always run executables with a leading `./`. Every time I do a shell PR I learn something new.

I also did not want to put the `abs/path` since that would require that you run the script from the `gRPC-go/` dir

Reply:
> for whatever reason i've always run executables with a leading `./`.

If you're running one in the current directory, then that's important.  If it's in an absolute path, you'd use `/usr/bin/ls`, e.g.  If it's in a relative path, then either `./bin/foo` or `bin/foo` work.